src/modules/launcher/Plugins/Microsoft.Plugin.Folder/Main.cs

```diff
@@ -234,10 +255,20 @@ private static List<Result> QueryInternal_Directory_Exists(Query query)
                 }
 
                 throw;
-            }
-
-            // Initial ordering, this order can be updated later by UpdateResultView.MainViewModel based on history of user selection.
-            return results.Concat(folderList.OrderBy(x => x.Title)).Concat(fileList.OrderBy(x => x.Title)).ToList();
+            }
+
+            if(folderList.Count > _settings.MaxFolderResults || fileList.Count > _settings.MaxFileResults)

```

Comment:
Should we make these values modifiable by the user by adding them to the settings page? (@crutkas, @saahmedm, @ryanbodrug-microsoft)

Reply:
Not right now, I think we can push this PR in and have a separate issue to exhibit interest